admin-site/src/EventsList.svelte

```diff
@@ -56,19 +63,33 @@
   }
 
 
+
 </script>
+<style>
+ .eventLable {
+  word-wrap:break-word;
+ }
+ h3 {
+   font-size: 1.55rem;
+ }
+ 
+</style>
 
 <Array
-      arrayName = {events} 
-      emptyArrayMessage = "This happens on occasion. No occasions for this event yet.">
-        {#each events as item (item.id)}
-          <div class="row mt-2">
-            <div class="col-6 text-right">
-                <h3>{item.label}:</h3>
-            </div>
-            <Button on:click={()=> eventButton(item.label)}
-              buttonHtml='<p class="mb-0">Occasions&nbsp;<span style="font-size: 1.1rem; vertical-align: middle" class="fas fa-angle-right" /></p>'
-              gridStyle ="col-6" />
-          </div>
-        {/each}
-</Array>
\ No newline at end of file
+  arrayName = {events} 
+  emptyArrayMessage = "This happens on occasion. No occasions for this event yet.">
```

Comment:
It's not a high priority for this sprint, but we should pull all English-language strings used for error messages / status messages / etc into their own file and import & reference as necessary.

Reply:
Ok, cool. I'll keep that in mind